crates/c/src/lib.rs

```diff
@@ -90,6 +110,19 @@ impl WorldGenerator for C {
         self.world = name.to_string();
         self.sizes.fill(resolve);
         self.world_id = Some(world);
+
+        let mut interfaces = HashMap::new();
+        let world = &resolve.worlds[world];
+        for (key, _item) in world.imports.iter().chain(world.exports.iter()) {
+            let name = resolve.name_world_key(key);
+            interfaces.insert(name, key.clone());
+        }
+
+        for (from, to) in self.opts.rename.iter() {
+            if let Some(key) = interfaces.get(from) {
```

Comment:
Sound we consider printing warning here if the user specifies a rename that's not used?